mod/ckeditor/views/default/js/ckeditor/boot.js

```diff
@@ -0,0 +1,16 @@
+define(function (require) {
+	var $ = require('jquery');
+
+	return {
+		addBehavior: function (context) {
+			$longtexts = $('.elgg-input-longtext:not([data-cke-init])', context);
+			if ($longtexts.length) {
+				require(['elgg/ckeditor'], function(elggCKEditor) {
```

Comment:
Anyway the point of this is to conditionally avoid loading the module completely.